### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by SilencerWeb**
This PR brings updated README.

Few suggestions:
1. Let's split images and icons? Right now we store all SVG in the folder `svg` and images with all other extensions in the folder `images`, I suggest storing all images (not depending on file extension) in the folder `images` and icons in the folder `icons`.
2. Let's store all reusable icons in the folder `icons`, kinda curious how we don't have this folder yet
3. Update LICENSE. Currently, copyright is under "gatsbyjs", is it correct or should we change it somehow? Or maybe we can get rid of this file?
4. Restructure SASS mixins as we did in other projects because right now they are all stored in one file

### CODE REVIEW

**Comment by lnikell** [comment]
1. There is a problem since we usually try to store svg near with the component it was used, however, rasterized images to use it with gatsby image we have to store in images folder
2. @BolotskyDev what do you think? I'm not against this idea but just want to be careful adding new folders into the project.
3. We can get rid of it.

File: README.md [hunk 1]
```diff
@@ -1,97 +1,174 @@
-<!-- AUTO-GENERATED-CONTENT:START (STARTER) -->
-<p align="center">
-  <a href="https://www.gatsbyjs.org">
-    <img alt="Gatsby" src="https://www.gatsbyjs.org/monogram.svg" width="60" />
-  </a>
-</p>
-<h1 align="center">
-  Gatsby's default starter
-</h1>
+# PixelPoint Gatsby Starter
```
Comment (lnikell) [inline README.md]: Pixel Point not PixelPoint :)

File: README.md [hunk 2]
```diff
@@ -1,97 +1,174 @@
...
-## 🚀 Quick start
+2. Install dependencies
 
-1.  **Create a Gatsby site.**
+```bash
+npm install
+```
 
-    Use the Gatsby CLI to create a new site, specifying the default starter.
+3. Copy .env.example and rename it into .env
```
Comment (lnikell) [inline README.md:32]: Copy .env.example and rename it into .env. You can also fill there all remaining env variables required for the project.

Maybe we can extend it this way

**Comment by bolotskydev** [comment]
Separating `icons` makes sense to me, since in current setup we are reusing them by copypasting from one local `svg` to another local `svg` folder, which is suboptimal. However, I don't think that storing _all_ images and `svg` illustrations in upper-level `images` folder is a way to go - it is much more convenient to manage block/component-scoped assets locally. 

Given that, I guess we could do the following: 
- create high level `icons` folder for storing reusable icons
- store reusable images and svg illustrations in high level `images` folder 
- manage one-off case assets locally in `component/images` folder no matter what extension they have, `svg|jpeg|png` - after moving icons away they roughly will have the same purpose

File: README.md (hunk 2 above)
Comment (SilencerWeb) [reply README.md:32]: Good idea but our current env variables do not require any change after copying .env.example

**Comment by SilencerWeb** [comment]
> store reusable images and svg illustrations in high level `images` folder

Can't really see how we can achieve this, how would the folder look like inside? How would we name images?

Also, I don't like the idea of storing in `images` any images that are not being used using `gatsby-image` because `images` is a source of `gatsby-source-filesystem` and not sure if it is a good idea to mix up images that are being used using `gatsby-images` and images that are being regularly imported in a component or styles but yeah, unfortunately, I don't see a better option here to reuse images. But still can't picture the structure and how we would name them.

> manage one-off case assets locally in component/images folder no matter what extension they have, svg|jpeg|png - after moving icons away they roughly will have the same purpose

I like this idea but I don't like the idea of moving ALL icons to `icons` even if they are not being reused. I think we can do it only if our designers would have a library of icons so we could have it too to make sure there won't be cases when we have to come up with a stupid name for 10th arrow icon in the project (e.g. arrow, arrow-small, arrow-large, arrow-outlined, arrow-thick, arrow-thin, etc because it is just confusing).

**Comment by lnikell** [comment]
@SilencerWeb I merged it since it's already good enough to be in master, all other opened discussions we can leave for later.

**Pull request merged**